comfy_cli/command/run/preflight.py

```
@@ -0,0 +1,157 @@
+        return json.loads(body)
+    except json.JSONDecodeError as e:
+        renderer.error(
+            code="object_info_unavailable",
+            message="Server returned invalid JSON for /object_info",
+            hint="check that the host:port really is a ComfyUI server",
+            details={"status": 200, "body": body.decode("utf-8", errors="replace")[:_MAX_BODY_PREVIEW]},
+        )
+        raise typer.Exit(code=1) from e
+
+
+def _preflight_validate(renderer, workflow: dict, object_info: dict, *, target_label: str = "server") -> None:
```

Pylint (4.0.5)

[warning] Unused argument 'target_label'

(W0613)